Menu/MenuBuilder.php

```diff
@@ -148,14 +137,4 @@ public function createSidebarMenu()
 
         return $event->getMenu();
     }
-
-    /**
-     * Sets the request the service.
-     *
-     * @param Request $request
-     */
-    public function setRequest(Request $request = null)
```

Comment:
Please keep the method for the next minor release and create a new PR for the next major version.